spec/factories/areas.rb

```diff
@@ -0,0 +1,6 @@
+FactoryGirl.define do
+  factory :area do
+    name "池袋"
+    geohash "xn7770"
```

Comment:
__[rubocop]__
* Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)